lib/phlex/tag.rb

```diff
@@ -23,11 +23,11 @@ def attributes=(attributes)
     def classes=(value)
       case value
       when String
-        @classes << value.prepend(SPACE)
+        @classes << SPACE << value
       when Symbol
-        @classes << value.to_s.prepend(SPACE)
+        @classes << SPACE << value.to_s
       when Array
-        @classes << value.join(SPACE).prepend(SPACE)
+        value.each { @classes << SPACE << _1 }
```

Comment:
`String#<<` is faster than `Array#join` and will be even faster in future Rubies.